bacon/model/Transaction.swift

```diff
@@ -146,39 +146,32 @@ class Transaction: Codable, Observable {
 // MARK: Transaction: Equatable
 extension Transaction: Equatable {
 
-    static func == (lhs: Transaction, rhs: Transaction) -> Bool {
-        return lhs.date == rhs.date
-            && lhs.type == rhs.type
-            && lhs.frequency == rhs.frequency
-            && lhs.category == rhs.category
-            && lhs.amount == rhs.amount
-            && lhs.description == rhs.description
-            && lhs.location == rhs.location
-            && lhs.image?.image.pngData()?.base64EncodedString() == rhs.image?.image.pngData()?.base64EncodedString()
+    /// Compares 2 transactions.
+    /// - Returns: true if they have equal properties.
+    func equals(_ transaction: Transaction) -> Bool {
+        return date == transaction.date
+            && type == transaction.type
+            && frequency == transaction.frequency
+            && category == transaction.category
+            && amount == transaction.amount
+            && description == transaction.description
+            && location == transaction.location
+            && image?.image.pngData()?.base64EncodedString() == transaction.image?.image.pngData()?.base64EncodedString()
     }
 
-    static func != (lhs: Transaction, rhs: Transaction) -> Bool {
-        return !(lhs == rhs)
+    static func == (lhs: Transaction, rhs: Transaction) -> Bool {
+        return lhs === rhs
```

Comment:
Perhaps you wanna leave a comment explaining why we do this? So that when the graders see they won't go 'wtf?'

Reply:
HAHA okay I will.

Reply:
Updated with comments @GilgameshTC